docs/src/pages/landing-page/QuasarComponents.vue

```diff
@@ -180,4 +182,11 @@ export default defineComponent({
   }
 }
 
+.slide-fade-enter-active {
```

Comment:
Not sure what this should do, but it doesn't seem it's doing anything 🤔

Reply:
It's for the ease in out animation of cards. It's actually doing something (if you increase the transition time for instance, you can see it transition slower), and overall it behaves a lot less clunky than the FadeIn Out.

Reply:
Ok, let's leave this as is, but we need to revisit this again, to make it look "cooler" in some way
We'll need to ask Nick what he wants on this regard